### PR TITLE
Test/auction gen & lock token

### PR DESCRIPTION
**Pull request by Velenir**
- Add test for `LockingToken4Reputation`
- Add test for `Auction4Reputation`

### CODE REVIEW

File: contracts/whitelisting/BasicTokenWhitelist.sol [hunk 1]
```diff
@@ -2,6 +2,7 @@ pragma solidity ^0.4.24;
 
 import "@gnosis.pm/dx-contracts/contracts/base/TokenWhitelist.sol";
 
+
 contract BasicTokenWhitelist is TokenWhitelist {
     constructor () public {}
 }
```
Comment (orenyodfat) [inline contracts/whitelisting/BasicTokenWhitelist.sol]: please remove CR

Comment (Velenir) [reply contracts/whitelisting/BasicTokenWhitelist.sol]: Can you point out where you see carriage return (`\r`)? According to search there isn't one in the whole project. Files should also be normalized with LF endings in the repo thanks to `.gitattributes`

Comment (orenyodfat) [reply contracts/whitelisting/BasicTokenWhitelist.sol]: screen shot 2019-01-22 at 15 12 34

Comment (Velenir) [reply contracts/whitelisting/BasicTokenWhitelist.sol]: Do you mean LF(`\n`) on line 5? Not carriage return?

Comment (orenyodfat) [reply contracts/whitelisting/BasicTokenWhitelist.sol]: line 8 in the image above

Comment (Velenir) [reply contracts/whitelisting/BasicTokenWhitelist.sol]: That is very strange because I simply don't see it.
Fixed(?) by editing here on github.

**Pull request merged**